.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - '*.md'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
```

Comment:
Why do we need this?

Reply:
For multi-architecture images, I haven't specified the platforms in the build step. Which ones should I build for?

Reply:
I'd say for all commons (darwin, linux, arm, windows).

Reply:
Done

Reply:
as you now only build for linux, can be removed again, right?

Reply:
Yes

Reply:
Removed and tested